src/components/views/DepartmentPreview/index.tsx

```diff
@@ -13,7 +13,7 @@ class Preview extends Component<Tprops, Tstate> {
     super(props);
 
     const { initialSelected } = this.props;
-
+    console.log(mock);
```

Comment:
Remove the console log and import of `mock`.

Reply:
done